shuttle/db/db_functions.py

```diff
@@ -213,3 +213,22 @@ def get_shuttle_logs():
     sql = "SELECT * FROM SHUTTLE_DRIVER_LOGS ORDER BY LOG_DATE"
     results = query(sql, 'read')
     return results
+
+
+# Method that grabs the last data that was inserted into the database. This assumes the latest data has the largest id
+def get_last_location():
```

Comment:
Something to consider is that there are multiple drivers. So there are multiple relevant "last" locations. Maybe its a bit early to consider this, but I figured I would comment

Reply:
Yeah this is a good comment since we will potentially have multiple drivers marking different locations (like on call shuttle vs normal shuttle).